### DISCUSSION TITLE
Indexing with more than 1 computer

### DISCUSSION THREAD

**Discussion by paulobreim** [sepinf-inc/IPED#2045] [Q&A]
1 - Suppose I have a case, already indexed, and I want to add other images to this case (option --append).
2 - Suppose I have 2 computers and in this case it is on a HD that both computers have access to.
3 - Can I run an IPED on each computer, having the output in the same location?

**Comment by lfcnassif** [comment]
It's not possible, the application wasn't designed to do this.

**Comment by paulobreim** [comment]
tks
